README.adoc

```diff
@@ -137,6 +137,13 @@ the working directory with the contents:
 {:drawbridge {:http-headers {:Authorization "Bearer <JWT token>"}}}
 ----
 
+Also, for using connecting repl via endpoint make sure that you added
```

Comment:
It seems to me that this should be in "Usage", maybe under a "Setup" sub-section.